app/DAL/repositories.py

```diff
@@ -33,6 +34,12 @@ def add_dog_with_images(self, dogDTO: DogDTO) -> DogDTO:
                 return dogDTO
         except SQLAlchemyError as e:
             # Rollback transaction on error
+            logger.error(f"DB Error while adding dog with images: {e}")
```

Comment:
@YanDav78 Can you change that to `logger.exception` so we have the stack trace automatically?

Reply:
Done